### PR TITLE
 Created landscape mode for pgame Activity.

### PR DESCRIPTION
**Pull request by yatna**
The landscape mode has been created for pgame Activity .
Here is what it looks like on a 4.5" mobile screen.

screenshot_2016-01-28-04-05-27

### CODE REVIEW

File: PowerUp/app/src/main/res/layout-land/pgame.xml [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            android:layout_alignParentRight="true"
+            android:layout_marginTop="@dimen/power_bar_margin_top"
+            app:rcBackgroundColor="@android:color/black"
+            app:rcProgressColor="@color/powerup_purple_light"
+            app:rcProgress = "@integer/half_progress"
+            app:rcMax="@integer/max_progress"
+            android:layout_alignParentTop="true" />
+        <com.akexorcist.roundcornerprogressbar.RoundCornerProgressBar
+            android:layout_width="@dimen/power_bar_width"
+            android:id="@+id/powerbarStrength"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout-land/pgame.xml]: Declare id as first attribute for the item

File: PowerUp/app/src/main/res/layout-land/pgame.xml [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+            android:layout_marginTop="@dimen/power_bar_margin_right"
+            android:layout_alignParentRight="true"
+            app:rcBackgroundColor="@android:color/black"
+            app:rcProgressColor="@color/powerup_pink"
+            app:rcMax="@integer/max_progress"
+            app:rcProgress = "@integer/half_progress"
+            android:layout_below="@id/powerbarHealing" />
+        <com.akexorcist.roundcornerprogressbar.RoundCornerProgressBar
+            android:layout_width="@dimen/power_bar_width"
+            android:id="@+id/powerbarTelepathy"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout-land/pgame.xml]: Same

File: PowerUp/app/src/main/res/layout-land/pgame.xml [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+            app:rcProgressColor="@color/powerup_blue_green"
+            android:layout_marginRight="@dimen/power_bar_margin_right"
+            app:rcMax="@integer/max_progress"
+            android:layout_alignTop="@id/powerbarHealing"
+            android:layout_toLeftOf="@id/powerbarHealing"
+            android:layout_toStartOf="@id/powerbarHealing"
+            app:rcProgress = "@integer/half_progress" />
+        <com.akexorcist.roundcornerprogressbar.RoundCornerProgressBar
+            android:layout_width="@dimen/power_bar_width"
+            android:id="@+id/powerbarInvisibility"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout-land/pgame.xml]: Same

File: PowerUp/app/src/main/res/layout-land/pgame.xml [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentStart="true" />
+
+    <RelativeLayout
+        android:layout_width="@dimen/asker_image_view_width"
+        android:layout_height="@dimen/asker_image_view_height"
+        android:layout_alignTop="@id/questionView"
+        android:layout_alignParentRight="true"
+        android:layout_alignParentEnd="true"
+        android:id="@+id/relativeLayout">
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout-land/pgame.xml]: Same

File: PowerUp/app/src/main/res/layout-land/pgame.xml [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+            android:layout_alignParentStart="true" />
+
+        <ImageView
+            android:id="@+id/clothImageView"
+            android:layout_width="@dimen/cloth_view_width"
+            android:layout_height="@dimen/cloth_view_height"
+            android:layout_alignTop="@id/eyeImageView"
+            android:layout_centerHorizontal="true"
+            android:src="@drawable/cloth1" />
+
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout-land/pgame.xml]: Don't leave these empty lines

File: PowerUp/app/src/main/res/values/dimens.xml [hunk 6]
```diff
@@ -51,7 +51,7 @@
     <dimen name="powerup_icon_width">150dp</dimen>
     <dimen name="powerup_width">304dp</dimen>
     <dimen name="power_bar_view_height">75dp</dimen>
-    <dimen name="game_text_size">20dp</dimen>
+    <dimen name="game_text_size">17dp</dimen>
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/values/dimens.xml]: Don't change the game text size as this will affect the entire application

**Comment by yatna** [comment]
1.) Id has been declared as the first attribute.
2.) Blank lines have  been removed.
3.) text_size has been restored to 20dp.

File: PowerUp/app/src/main/res/values-land/dimens.xml [hunk 7]
```diff
@@ -37,5 +37,6 @@
     <dimen name="avatar_room_width">80dp</dimen>
     <dimen name="avatar_room_height">80dp</dimen>
     <dimen name="avatar_room_continue_button_land">30dp</dimen>
+	<dimen name= "completed_game_textview">50dp</dimen>
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/values-land/dimens.xml]: Please ident everything correctly in all files

File: PowerUp/app/src/main/res/layout-land/pgame.xml [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+        android:background="@android:color/white"
+        android:backgroundTint="@color/abc_secondary_text_material_dark"
+        android:ems="@integer/ems"
+        android:textColor="@android:color/black"
+        android:textSize="@dimen/game_text_size"
+        android:layout_below="@id/scenarioNameEditText"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentStart="true" />		
+    <RelativeLayout
+	    android:id="@+id/relativeLayout"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout-land/pgame.xml]: Please ident this correctly

**Comment by alma-castant** [comment]
Hi @yatna what happens if the boyfriend's text is longer? You may want to make the boxes and text smaller (only here for this configuration, not for the entire application). In the long term, the layout file will need to be refactored to put elements such as the header, the footer and Rosie into separate xml files. Consider whether you want to do this now

**Comment by yatna** [comment]
All files have been intended properly now.

**Comment by yatna** [comment]
I totally agree with you @alma . I am planning to include this change in my GSoC 2016 proposal.The pgame layout file has become quite bulky with a lot of elements . Thus we can define its various elements into different layout files, and in the main pgame file , we can just use those layouts. This needs to be done for both landscape and portrait mode.

**Comment by chhavip** [comment]
Fixed in GSoC 2016

**Pull request closed without merging**